src/lib/comms/sol-http-server-impl-microhttpd.c

```diff
@@ -231,12 +236,33 @@ headers_iterator(void *data, enum MHD_ValueKind kind, const char *key, const cha
 }
 
 static int
+get_static_file(const char *dir, const char *url)
+{
+    int ret;
+    char path[PATH_MAX];
+
+    /* url given by microhttpd starts from /. For instance,
+     * https://www.solettaproject.com => url == /
```

Comment:
URLs might begin with more than one `/`. Not much of a problem (`GET //////////////index.html` should serve `index.html`), but it's unusual and pretty cheap to sanitize.